omod/src/test/java/org/openmrs/module/coreapps/fragment/controller/visit/ParserEncounterIntoSimpleObjectsTest.java

```diff
@@ -297,6 +300,29 @@ public void testParsingSimpleObs() throws Exception {
 		assertThat(path(parsed.getObs(), 0, "answer"), is((Object) valueText));
 	}
 
+	@Test
+	public void testParsingNumericDataTypeWithAllowDecimalAsFalseToSimpleObs() throws Exception {
+		ConceptDatatype numericDatatype  = new ConceptDatatype() ;
+		numericDatatype.setName("Numeric");
+		numericDatatype.setHl7Abbreviation("NM");
```

Comment:
Do you need this?

Reply:
Yes, I need, if I don't put that I will have NPE.